main.py

```diff
@@ -16,6 +19,12 @@
 from prometheus_client.parser import text_string_to_metric_families
 from pySMART import Device
 
+tegrastats_queue = LifoQueue()
```

Comment:
Any reason for a LifoQueue rather than a plain (Fifo) Queue? It shouldn't matter since you process the whole thing each time anyway. Just curious.

Reply:
Oh I was originally only pulling the latest.. but I changed that and forgot to change off of the Lifo